state/logs.go

```diff
@@ -445,7 +440,7 @@ func (t *logTailer) processReversed(query *mgo.Query) error {
 		return errors.Errorf("too many lines requested (%d) maximum is %d",
 			t.params.InitialLines, maxInitialLines)
 	}
-	query.Sort("-e", "-t", "-_id")
+	query.Sort("-t", "-_id")
```

Comment:
Yeah, we no longer store the model UUID (what was "e") as part of the record. Each model's logs now go into a separate collection.